eth_abi/exceptions.py

```diff
@@ -6,88 +6,67 @@ class EncodingError(Exception):
     Base exception for any error that occurs during encoding.
     """
 
-    pass
```

Comment:
Do we still need these `pass`es?

Reply:
They're removed by autoflake - https://github.com/PyCQA/autoflake - which says it removes "useless" `pass` statements.

A docstring renders a `pass` unnecessary: https://stackoverflow.com/questions/49754137/empty-class-with-comment-same-as-pass

I left it on default settings, but can put'm back if you want :smile:

Reply:
Oh cool. I'm all for default settings! 🚀